colors.md

```diff
@@ -14,40 +15,48 @@ violet:    #6c71c4;
 blue:      #268bd2;
 cyan:      #2aa198;
 green:     #859900;
+```
 
-ACCENT:
-#b58900 > instance?
-#cb4b16 > include/import
-#dc322f > support function, escape
-#d33682 > TODO
-#6c71c4 > md link, number
-#268bd2 > variable, tag
-#2aa198 > string
-#859900 > punctuation.definition
+## ACCENT
+Colour    |  Use 
+--------- | -----
+`#b58900` | instance?
+`#cb4b16` | include/import
+`#dc322f` | support function, escape
+`#d33682` | TODO
+`#6c71c4` | md link, number
+`#268bd2` | variable, tag
+`#2aa198` | string
+`#859900` | punctuation.definition
 
 
-LIGHT:
-base3   > bg
-base2   > highlights bg (current line, gutter)
-base1   > comments
-base0   > unused
-base00  > default
-base01  > emphasis
-base02  > caret
-base03  > unused
+## LIGHT
+Colour |  Use 
+------ | -----
+base3  | bg
```

Comment:
It's used for the selection area background, where instead of picking a solid color from the list of "official colors" a mix of two colors is used to simulate transparency.